.gitignore

```diff
@@ -1,4 +1,9 @@
-dist
+# Ignore
+dist/webport/*
+dist/src/*
 node_modules
 .DS_Store
-coverage
\ No newline at end of file
+coverage
+
+# Don't ignore
+!dist/webport/index.html
```

Comment:
This should in fact be ignored. We don't need build files in the repository. We get auto releases by every PR. They are kept outside the repo. Can you remove this line?

Reply:
I had commented out but I apparently forgot to push last night, my bad!